src/tokens/solana.tokenlist.json

```diff
@@ -174483,7 +174483,7 @@
       "symbol": "SHP",
       "name": "ShipHero",
       "decimals": 0,
-      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/5XFJAEyKtGUruRnjwzKZzDVwjzyyBtLw5TUhyZKyqUbk/logo.png",
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/8C8YZ1BqU2Lu4wQZcboc7znD6axPdLdU2iTi91nauVo4/logo.png",
```

Comment:
@keone Not sure if you're still around (I hope you are and doing well)
You helped me out last time, can you please do it again. (diff will not be merged without explicit approve).
Thank you very much!